pkg/kfapp/kustomize/kustomize.go

```diff
@@ -350,36 +346,84 @@ func (kustomize *kustomize) Delete(resources kftypesv3.ResourceEnum) error {
 				Message: msg,
 			}
 		}
+	}
```

Comment:
I wonder if line 339 to line 349 are still needed when line 361 below already handles the empty case.

Reply:
Nice catch, you're right!